docker/entrypoint.sh

```diff
@@ -14,7 +14,4 @@ while ! nc -z "${DATABASE_HOST}" "${DATABASE_PORT}" &> /dev/null; do
     sleep 1;
 done
 
-# Initialize migrations and SETA
-./initialize_data.sh
-
```

Comment:
@armenzg I can not get the `backend` image to startup with this line.

https://bugzilla.mozilla.org/show_bug.cgi?id=1612977

Reply:
@armenzg This seems important. I will wait for the bug to be fixed because `master` is no longer usable to me.